documentation/concepts/check.md

```diff
@@ -4,25 +4,65 @@ menu: check
-- A temporary database is used and your migrations are run against this to produce a model of your target configuration. 
-    - There is no requirement for the temporary database to be in your production system 
-    - Please note that the temporary database **will be cleaned** before the operation starts
+In Flyway, “Checks” is the collective term we use for the pre- or post-deployment analysis of some aspect of your database migration. Checks are instantiated using the top level `check` command.  
 
-## Flags
-The command uses a flag to indicate the kind of report to generate:
+Before performing a deployment to the target database (most notably, production), you might want to look over what you’re about to do and understand one or more of the following: 
 
-### `-changes` 
+- Does this set of changes affect the objects I expect it to, or will I be inadvertently having an impact on something else? 
+- What database changes have been made recently, that coincide with the changes in database performance we are seeing? Are the two related? 
+- Is the production database in the same state you were expecting when I began developing my changes? Has anything about the target database changed that would mean my changes no longer have the desired effect?  
+- Does our approach to database change development meet our internal policies? Are our migration scripts adhering to our naming conventions, for example? Are we following the security best-practices required of us by our external regulatory requirements? 
 
-This is used to report on the pending changes that have yet to be applied to the database. You might use this as part of an in-person review process or as a validation of the effect of all pending changes.
-- The command then runs a comparison engine to identify differences between the temporary database at two points:
-    - The state equivalent to the current version of the target database
-    - The state once all pending migrations have been run
+Each of these scenarios can be met with the `check` command, using the corresponding flag: 
+
+| Scenario                                                           | Command & Flag     | Output                                  |
+|--------------------------------------------------------------------|--------------------|-----------------------------------------|
+| _In beta_<br>Will these changes have the effect I am expecting?    | check **-changes** | ChangeReport.html,<br>ChangeReport.json |
+| _In beta_<br>What database changes have been made recently?        | check **-changes** | ChangeReport.html,<br>ChangeReport.json |
+| _Coming soon_<br>Is the production database in the state I am expecting it to be in | check **-drift** |                          |
+| _Coming soon_<br>Are our changes following internal policies                        | check **-drift**                            |
```

Comment:
The last two rows look like they might be missing a `?` in the `Scenario` column?

The last row seems to be missing a `|` at the end of the `Command & Flag` column

Could the first two rows have more whitespace at the end of the first column so that all three columns are aligned in source? It shouldn't affect the way it renders, but would make the source more readable